api-reference/user-app/vitals/vital-information-table.mdx

```diff
@@ -0,0 +1,20 @@
+Here is the information regarding various vitals:
+
+| **Vital Name**                                    | **LOINC Code** | **Measurement Unit**      | **MCUM UCUM Code**   | **EKA ID**        |
+|:--------------------------------------------------|:---------------|:--------------------------|:----------------------|:-------------------|
+| **Min Heart Rate**                              |                | BPM (beats per minute)     |                      |                   |
+| **Max Heart Rate**                              |                |                           |                      |                   |
+| **Mean Heart Rate**                             |                |                           |                      |                   |
+| **Random Blood Glucose**                        | 74774-1        | mg/dL                     |                      | lb-7881503220     |
+| **Fasting Blood Glucose (overnight)**            | 1558-6         | mg/dL                     |                      | lb-5349691605     |
+| **After Food Blood Glucose (>2hr after food)**   | 1521-4         | mg/dL                     |                      | lb-5528009855     |
+| **Blood Oxygen**                                | 2708-6         | %                         |                      | lb-1771358750     |
```

Comment:
This is streaming data too so add the mean, min , max for this too along with base `Blood Oxygen` if available